superset-frontend/src/utils/downloadAsImage.ts

```diff
@@ -77,7 +71,7 @@ export default function downloadAsImage(
     return domToImage
       .toJpeg(elementToPrint, {
         quality: 0.95,
-        bgcolor: GRAY_BACKGROUND_COLOR,
+        bgcolor: supersetTheme.colors.primary.light3,
```

Comment:
```suggestion
        bgcolor: supersetTheme.colors.grayscale.light4,
```
This is the small change I'm blabbing about in the git history investigation :P

Reply:
Yeah, that makes sense to me! Since #f5f5f5 wasn't in the theme I wasn't sure which was the correct color to choose here. Thanks for the point in the right direction! Corrected the color in [`this commit`](https://github.com/apache/superset/pull/22011/commits/34ab7e753991aa3f61d5f13819bb7db70ced9a42).